examples/points/main.js

```diff
@@ -52,28 +52,28 @@ $(function () {
   });
 
   // Create an osm layer with custom tile url for a white background.
-  map.createLayer(
-    'osm',
-    {
-      url: function () {
-        return 'white.jpg';
-      }
-    }
-  );
+  // map.createLayer(
```

Comment:
Can you make this a new example rather than modifying the existing points example?

Reply:
Yep, I need to clean this up.